python-phoenixdb/phoenixdb/avatica/client.py

```diff
@@ -168,7 +169,7 @@ def _post_request(self, body, headers):
         if not self.session:
             logger.debug("Creating a new Session")
             self.session = requests.Session()
-            self.session.headers.update(headers)
+            self.session.headers.update({**self.headers, **headers})
```

Comment:
We can just set the pre-merged set ofe headers here.

Reply:
Done